protos/feast/core/CoreService.proto

```diff
@@ -153,4 +178,34 @@ message UpdateStoreResponse {
     }
     feast.core.Store store = 1;
     Status status = 2;
+}
+
+// Request to create a project
+message CreateProjectRequest {
+    // Name of project (required)
+    string name = 1;
+}
+
+// Response for creation of a project
+message CreateProjectResponse {
+}
+
+// Request for the archival of a project
+message ArchiveProjectRequest {
```

Comment:
What happens to the features ingested when a project is archived?

Reply:
It continues to work. Ingestion is unaffected.